### PR TITLE
movie: Wing Commander (1999) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
An unimaginative, 20 years too late Star Wars/Star Trek inspired space opera type film. Rating 1/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-1990s.st.txt [hunk 1]
```diff
@@ -11655,6 +11655,50 @@ giant turtle [Gamera.]
...
+movie: Wing Commander (1999)
+============================
+
+:: Title
+Wing Commander
+
+:: Description
+The year is 2654, and it is up to a crop of fresh Terran Confederation
+military recruits to thwart the efforts of the cat-like Kilrathi aliens from
+exterminating the human race. It is loosely based on the video game series of
```
Comment (odinlake) [inline notes/stories/film/film-scifi-1990s.st.txt:11667]: A confusing change of vantage point in these two sentences. (Just saying)

File: notes/stories/film/film-scifi-1990s.st.txt [hunk 2]
```diff
@@ -11655,6 +11655,50 @@ giant turtle [Gamera.]
...
+:: References
+https://en.wikipedia.org/wiki/Wing_Commander_(film)
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+interstellar war [The Terran Confederation and the Kilrathi Empire were fighting a Milky way galaxy spanning war.]
+space colonization [A premise of the story is that the human race had reached out to colonize large swaths of the galaxy by the year 2654.]
+extraterrestrial being [The cat-like Kilrathi aliens were seeking the complete eradication of the human race.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-1990s.st.txt:11682]: https://www.themeontology.org/theme.php?name=felinoid%20being ?!

**Review by odinlake — APPROVED** [review]
I see, felinoid is minor but extraterrestrial being is major. Well fair enough.

File: notes/stories/film/film-scifi-1990s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-1990s.st.txt:11682]: Ah, that one is tossed in as minor on account that they were cat-like, but nothing was made of it to the extent where one wonders why they even bothered...

File: notes/stories/film/film-scifi-1990s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-1990s.st.txt:11667]: I agree. How about this:

"Set in the year 2654, the story follows a crop of fresh Terran Confederation military recruits in their effort to thwart a hostile alien empire from exterminating the human race. It is loosely based on the video game series of the same name."

**Comment by odinlake** [comment]
A most excellent description. I guess, in addition, I would go for boring
precision over sonorous flair and replace the remaining “It” with “The
film”.

**Pull request merged**